### DISCUSSION TITLE
Strings with a % in them (not a placeholder) are being turned into functions.

### DISCUSSION THREAD

**Discussion by oliverfoggin** [SwiftGen/SwiftGen#904] [Q&A] [answered]
I'm using the standard `structured-swift5` template in my project but I'm experiencing an issue with strings that have % signs in them. (Not placeholders, just % signs as part of the text).

When they are run through `swiftgen` it outputs them as functions with an input rather than as a static property.

e.g. this...

```
"example_text_description" = "30% example";
```

Will get turned into something like this...

```
// other nested enums...
public static func description(_ p1: Float) -> String {
  return L10n.tr("Localizable", "example_text_description", p1)
}
```

Please could you let me know if there is a way to stop this from occurring.

Thanks

**Comment by djbe** [comment, marked as answer]
Yes, you have to escape your `%`, by changing it to `%%`.

It's because `% e` is a valid pattern for formatting a `Float`. See also #460 #577 #859 (and probably a bunch more).
